### DISCUSSION TITLE
How to connect the local ssh without network?

### DISCUSSION THREAD

**Discussion by LaSeineJ** [Eugeny/tabby#5447] [Q&A]
I`m using Tabby 1.0.169 on win11. When I`m online, I can connect my local ssh in the virtual machine，but I can`t connect it without network.

**Comment by Eugeny** [comment]
Are you able to connect with to the same VM IP when using a different terminal? (e.g. `ssh ...` in Powershell)?

**Comment by LaSeineJ** [comment]
Yeah,&nbsp; I can connect the same ssh with mobaxterm.

---Original---
From: ***@***.***&gt;
Date: Fri, Jan 14, 2022 07:24 AM
To: ***@***.***&gt;;
Cc: ***@***.******@***.***&gt;;
Subject: Re: [Eugeny/tabby] How to connect the local ssh without network? (Discussion #5447)




 
Are you able to connect with to the same VM IP when using a different terminal? (e.g. ssh ... in Powershell)?
 
—
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.
